Export micromega (#15921)
coq/coq: doc/sphinx/addendum/micromega.rst

.. tacn:: lra_Q @ltac_expr
          lra_R @ltac_expr

>           lra_wit_Q @ident @one_term

Reviewer: It's a trifle odd to have a change log entry announcing something that's for internal use only.  You might mention that it's for internal use only in the changelog.  Or better, add a real description with examples.

   It performs a pre-processing similar to :tacn:`nra`. The obtained goal is
   solved using the linear integer prover :tacn:`lia`.

.. tacn:: xnia @ltac_expr

>    For internal use only (could change without deprecation phase).

Reviewer: ```suggestion
   For internal use only (it may change without notice).
```

   Goal forall x, -x^2 >= 0 -> x - 1 >= 0 -> False.
   intro x.
   psatz Z 2.

>    Qed.

Reviewer: Maybe use `.. coqtop::` instead to show the output if it would be helpful to the reader

Author: Unfortunately, as the comment above says, this requires the external tool CSDP which will cause issues compiling the doc. Fortunately enough, the only output is the fact that the goal is proved, so not much is lost.
